packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -161,6 +163,14 @@ const Panel: React.FC<SelectPanelProps> = ({
     [internalOpen],
   )
 
+  // used in SelectPanel.SearchInput
+  const moveFocusToList = () => {
+    const selector = 'ul[role=listbox] li:not([role=none])'
```

Comment:
The selector looks a bit too fragile, makes me a bit nervous 😬 but the query makes a lot of sense. Can we add tests/stories to make sure we always get the intended first element? (It is not a blocker, feel free to merge the PR)

Reply:
Will take it up in a new PR!